app/(aux)/settings.js

```diff
@@ -156,99 +469,392 @@ export default function Page() {
-							*Notify you when you have spend too much time on
-							Prayer Cirlce
-						</StyledText>
+						<StyledView className='w-[85%] items-center'>
+							<StyledText className='mt-3 text-[16px] font-bold text-center text-offwhite'>*Notify you when you have spend too much time on Prayer Cirlce</StyledText>
+						</StyledView>
+					</StyledView>
+				);
+			case 'hiddenPosts': // TODO: New version of post.js for hidden posts
+				return (
+					<StyledView className='flex-1 bg-grey py-3 items-center text-offwhite'>
+						<StyledView className='w-[90%] flex-1'>
+							<BottomSheetFlatList
+								data={hiddenPosts}
+								renderItem={({ item }) => (
+									<>
+										<Post
+											user={item[1].name}
+											img={item[1].profile_img}
+											title={item[1].title}
+											timestamp={item[1].timestamp}
+											content={item[1].text}
+											icon={item[1].type}
+											id={item[0]}
+											edited={item[1].edited}
+											comments={item[1].comments}
+											data={item[1]}
+										/>
+										<Button 
+											title='Unhide Post'
+											width='w-[70%]'
+											btnStyles='mt-2 mb-5 self-center'
+											press={() => unhidePost(item[0])}
+										/>
+									</> 
+								)}
+								keyExtractor={(item) => item[0]}
+								showsVerticalScrollIndicator={false}
+								ListEmptyComponent={() => (
+									<StyledView className='w-full justify-center items-center text-center'>
+										<StyledText className='font-bold text-[20px] text-offwhite text-center'>
+											No Posts Hidden!
+										</StyledText>
+									</StyledView>
+								)}
+							/>
+						</StyledView> 
+					</StyledView>
+				);
+			case 'passwordInfo':
+				return (
+					<StyledView className='flex-1 bg-grey items-center text-offwhite'>
+						<StyledView className='w-[85%] items-center'>
+							<StyledView className='flex-row justify-center'>
+								<StyledIcon name='warning-outline' size={30} color="#F9A826" className="w-[30px] h-[30px] mr-2"/>
+								<StyledText className='pt-1 text-[16px] font-bold text-center text-offwhite'>This cannot be reverted</StyledText>
+								<StyledIcon name='warning-outline' size={30} color="#F9A826" className="w-[30px] h-[30px] ml-2"/>
+							</StyledView>
+							<StyledView className='flex-row justify-center py-3'>
+								<StyledText className='mt-1 text-[16px] font-bold text-center text-offwhite'>Send a reset password to your email!</StyledText>
+								<StyledIcon name='mail' size={30} color="#FFFBFC" className="w-[30px] h-[30px] ml-2"/>
+							</StyledView>
+							<StyledText className='mt-3 text-[20px] font-bold text-center text-offwhite'>Password Rules:</StyledText>
+							<StyledText className='mt-3 text-[16px] font-bold text-center text-offwhite'>*Password must contain atleast: </StyledText>
+							<StyledText className='mt-3 text-[16px] font-bold text-center text-offwhite'>- 8 total characters </StyledText>
```

Comment:
```suggestion
							<StyledText className='mt-3 text-[16px] font-bold text-center text-offwhite'>- 8 characters </StyledText>
```
'total' is redundant because it already says 'must contain at least'